README.md

```diff
@@ -23,6 +23,15 @@ cd git-credential-msal
 python3 -m pip install .
 ```
 
+If you run into issues with the above, first verify that the above works with
```

Comment:
"the above" sounds a little weird twice in this sentence. Maybe "If you run into issues with the above, verify that the latest version of `pip` is installed." ?